docs/index.rst

```diff
@@ -2,20 +2,42 @@
 Welcome to qujax's documentation!
 =================================
 
-``qujax`` simulates quantum computation using `JAX <https://github.com/google/jax>`_.
+``qujax`` is a `JAX <https://github.com/google/jax>`_-based Python library for the classical simulation of quantum circuits. It is designed to be *simple*, *fast* and *flexible*.
 
-Source code can be found on `GitHub <https://github.com/CQCL/qujax>`_, including a suite of `example notebooks <https://github.com/CQCL/qujax/tree/main/examples>`_.
+It follows a functional programming design by translating circuits into pure functions. This allows qujax to `seamlessly and directly interface with JAX <https://jax.readthedocs.io/en/latest/notebooks/Common_Gotchas_in_JAX.html#pure-functions>`_
 
-The `pytket-qujax <https://github.com/CQCL/pytket-qujax>`_, extension is useful for converting a `tket <https://github.com/CQCL/pytket>`_ circuit into ``qujax``.
+Source code can be found on `GitHub <https://github.com/CQCL/qujax>`_, including a suite of `example notebooks <https://github.com/CQCL/qujax#examples>`_.
+
+The `pytket-qujax <https://github.com/CQCL/pytket-qujax>`_, extension can be used to directly translate a `tket <https://github.com/CQCL/pytket>`_ circuit directly into ``qujax``.
 
 Note that ``qujax`` assumes parameters are given in units of π (i.e. in [0,2] rather than [0, 2π]).
 
 Install
 =================================
-``qujax`` is hosted on `PyPI (aka the Cheese Shop) <https://pypi.org/project/qujax/>`_ and can be installed with
+``qujax`` is hosted on `PyPI <https://pypi.org/project/qujax/>`_ and can be installed with
+
+.. code-block:: bash
 
-``pip install qujax``
+   pip install qujax
```

Comment:
Why have you removed the `` ?

Reply:
I replaced it by the `.. code-block:: bash` directive just above, which in my opinion highlights it a bit better, this is what it looks like now:

![image](https://github.com/CQCL/qujax/assets/7880673/dbbc75a6-2815-49a0-8793-22570a616309)